wavefront/server/modules/voice_agents_module/voice_agents_module/controllers/tool_controller.py

```
@@ -0,0 +1,314 @@
+
+
+@tool_router.get('/v1/voice-agents/{agent_id}/tools')
+@inject
+async def get_agent_tools(
+    agent_id: UUID,
+    include_credentials: bool = Query(
+        False, description='Include real credentials (for internal services)'
+    ),
+    response_formatter: ResponseFormatter = Depends(
+        Provide[CommonContainer.response_formatter]
+    ),
```

Ruff (0.14.13)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)